wurst/systems/spells/SmokeMeat.wurst

```diff
@@ -21,6 +20,13 @@ init
                         charges += floor(maybeMeat.getCharges() / 2)
 
             if charges > 0
+                removeMeats(smoker)
                 let meat = createItem(ITEM_SMOKED_MEAT, smoker.getPos())
                 ..setCharges(charges)
-                UnitStackItem(smoker, meat)
\ No newline at end of file
+                UnitStackItem(smoker, meat)
+
+function removeMeats(unit smoker)
```

Comment:
Wouldn't this remove all meat that wasn't transformed?

Reply:
It does remove all meat, but all meat is transformed anyway in the function above.

Reply:
For some reason, I was thinking that only a limited amount of meat was transformed per casting.